api/docs/release.dox

```diff
@@ -241,6 +241,8 @@ Further non-compatibility-affecting changes include:
    trace analysis tool framework.
```

Comment:
> To preserve TRACE_MARKER_TYPE_FUNC_ markers related to functions with ID: 1, 2, 3, and 4.

Add "and remove the FUNC_ markers for all other ID values."

Reply:
Done.